codegen/protocol-test-codegen-local/Tests/swift-codegen/WaitersTests/WaiterModels+Equatable.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
Waiters does not have any code-generated tests.  So hard-coded Equatable types are provided for the models that Waiters tests need to compare.